packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec

```diff
@@ -48,6 +48,7 @@ Provides: foreman-plugin-%{plugin_name} = %{version}
 BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
 BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 4.14.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 5.0.0
```

Comment:
This is technically not necessary, and not present in the rpm/develop commit -- mind removing it?

Reply:
Not at all, done.